### PR TITLE
Issue #13213: Remove `//ok` comments from Input files `InputRequireThisTryWithResourcesOnlyOverlappingFalse`

### PR DESCRIPTION
**Pull request by abhishekmaity**
Towards #13213

1. Remove `//ok` only comments from `InputRequireThisTryWithResourcesOnlyOverlappingFalse.java`
2. removed corresponding `<suppress id="noCheckstyleInInputs">` from `checkstyle-input-suppression.xml` (not present in this branch)
3. Successfully passed `mvn clean verify`

### CODE REVIEW

**Comment by abhishekmaity** [comment]
@nrmancuso please review this PR and sorry for [#14539](https://github.com/checkstyle/checkstyle/pull/14539#issuecomment-1962986357).

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java [hunk 1]
```diff
@@ -20,22 +20,22 @@ public class InputRequireThisTryWithResourcesOnlyOverlappingFalse implements Aut
 
     void oneResource() {
         try (BufferedReader fldBufferedReader
-                     = new BufferedReader(new InputStreamReader(null, "utf-8"))) { } // ok
+                     = new BufferedReader(new InputStreamReader(null, "utf-8"))) { }
         catch (IOException e) { }
```
Comment (romani) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java:24]: I don't see removal of suppression in this PR.
Please recheck what you pushed

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java (hunk 1 above)
Comment (abhishekmaity) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java:24]: @romani done, please review PR

Comment (romani) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/InputRequireThisTryWithResourcesOnlyOverlappingFalse.java:24]: @abhishekmaity , please do not resolve review items, just reply "done" to each and every review items.

**Review by romani — APPROVED** [review]
Ok to merge if ci pass

**Comment by abhishekmaity** [comment]
> Ok to merge if ci pass

Please check now, only **Cirrus - JDK17** failing due to `Failed to start an instance: FAILED_PRECONDITION: Monthly compute limit exceeded!`

**Pull request merged**